samples/fabric/fabric-cli/makefile

```diff
@@ -3,10 +3,10 @@ build:
 	yarn link
 
 build-local:
-	cp package.json package-remote.json && \
+	(cp package.json package-remote.json && \
 		cp package-local.json package.json && \
-		npm install && \
-		mv package-remote.json package.json
+		npm install) || true
```

Comment:
Why is it OK for this to fail?

Reply:
Its not okay to fail, its just if it fails or succeeds, the `package.json` is reverted back in any case.
Otherwise we used to see `package.json` was remained to be a copy of `package-local.json`.